pages/EC-WebSphere_help.xml

```diff
@@ -6390,14 +6391,10 @@ A context root for each web module is defined in the application deployment desc
 
     <h1 id="releaseNotes">Release Notes</h1>
     
-    <h3>@PLUGIN_KEY@ 2.8.0</h3>
+    <h3>@PLUGIN_KEY@ 2.7.1</h3>
```

Comment:
Something strange is going on. Where is version 2.8.0?

Reply:
The minor version had been increased as a preparation to (FLOWPLUGIN-8206):" EC-WebSphere Create Configuration violates FLOWPLUGIN-7900". But later the one has been postponed and therefore these changes are not actual for now
https://github.com/electric-cloud/EC-WebSphere/pull/287